.circleci/config.yml

```diff
@@ -66,10 +66,5 @@ workflows:
           name: "Ruby << matrix.ruby_version >>, ES << matrix.elasticsearch_version >>"
           matrix:
             parameters:
-              ruby_version:
-                - 2.5.5
-                - 2.6.6
-                - 2.7.2
-              elasticsearch_version:
-                - 6.8.15
-                - 7.10.1
+              ruby_version: ["2.6.6"]
+              elasticsearch_version: ["6.8.15", "7.10.1"]
```

Comment:
Let's keep the original syntax, which I find cleaner and more readable; the behavior is equivalent. Also, we should continue to test 2.7.2 (which we may as well bump to 2.7.3) as well as 2.6.6 for forward compatibility.

Reply:
Changed.

Reply:
Changed.